src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -106,7 +104,7 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
 
     const overflowMenu: ThreeDotsMenuProps['menuItems'] = useMemo(
         () => [
-            ...(shouldShowEnterCredentials
+            ...(shouldShowEnterCredentials && (connectedIntegration === CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT || connectedIntegration === CONST.POLICY.CONNECTIONS.NAME.NETSUITE)
```

Comment:
This line introduced this bug (https://github.com/Expensify/App/issues/50357).
More context:  https://github.com/Expensify/App/issues/50357#issuecomment-2398155410